dev-client/src/screens/LocationScreens/components/soilId/SoilIdMatchesSection.tsx

```diff
@@ -79,9 +83,13 @@ const MatchTilesOrMessage = ({siteId, coords}: SoilIdMatchesSectionProps) => {
     case 'ready': {
       if (isSite) {
         return getSortedDataBasedMatches(soilIdData).map(dataMatch => (
-          <InfoOverlaySheet
+          <InfoSheet
             key={dataMatch.soilInfo.soilSeries.name}
-            Header={dataMatch.soilInfo.soilSeries.name}
+            heading={
+              <Heading variant="h4">
```

Comment:
These are the only two instances of an `InfoSheet` heading not being a `TranslatedHeading` component. I expect that in future typography work (https://github.com/techmatters/terraso-product/issues/909) we will find a way to combine `Heading` and `TranslatedHeading` (and other equivalents) to make specifying text content more standard / concise. I felt that it was acceptable to have two non-standard cases when we know we will revise our typography soon.

Reply:
Done.